rest/server_context.go

```diff
@@ -797,6 +797,21 @@ func (sc *ServerContext) initEventHandlers(dbcontext *db.DatabaseContext, config
 			return pkgerrors.Wrapf(err, "Error calling base.JSONUnmarshal() in initEventHandlers")
 		}
 
+		// Load Webhook Filter Function.
+		for _, event := range eventHandlers.DocumentChanged {
```

Comment:
We need to do this for all registered event handlers, not just DocumentChanged ones. They all have javascript filter functions.

Reply:
Included all registered event handlers (DocumentChanged, DBStateChanged)